src/haxeLanguageServer/helper/DocHelper.hx

```diff
@@ -105,6 +108,29 @@ class DocHelper {
 	public static function printCodeBlock(content:String, languageId:LanguageId):String {
 		return '```$languageId\n$content\n```';
 	}
+
+	public static function patchRange(doc:TextDocument, range:Range) {
```

Comment:
Moving this might make sense, but the name seems too generic for a utility class like this... At least I have a hard time telling what a `DocHelper.patchRange()` does exactly.

Reply:
I was tempted to rename it to `untrimRange`, also I wanted to use it in one other location, but ended up not using it....